src/resources/business-config.ts

```diff
@@ -13,6 +13,7 @@ export class Business {
   flows: Array<Flow>
 }
 
+// FUTURE: this object needs an interface or type
```

Comment:
Is FUTURE different from a TODO?

Reply:
For me, yes. (We have not previously discussed this as a group.)

TODO = a comment to myself to do this before I commit my changes (in scope of current ticket)

FUTURE = a comment to me/others that this should be looked at some time (not in scope of current ticket); sometimes I'll create a ticket for such items, sometimes not